pyanaconda/ui/gui/spokes/lib/subscription.py

```diff
@@ -100,3 +108,135 @@ def fill_combobox(combobox, user_provided_value, valid_values):
 
     # set the active id (what item should be selected in the combobox)
     combobox.set_active_id(active_id)
+
+    def clear_listbox(listbox):
```

Comment:
It looks like these functions are nested in the function `fill_combobox`. Is it necessary?

Reply:
Yeah, if it'S used just in a single place, we can drop the function and just include it.

Reply:
That is not necessary, I'm just not sure how it works. The functions `add_attached_subscription_delegate` and `populate_attached_subscriptions_listbox` are nested in the function `fill_combobox`, but `fill_combobox` doesn't call them.

Reply:
Oh, now I  see it! Yeah, the indentation is totally wrong - but given that it was wrong for all the functions I've added, I have completely missed that. :P Should be fixed now.